### DISCUSSION TITLE
DoT on iOS not working

### DISCUSSION THREAD

**Discussion by bloodyburger** [AdguardTeam/AdGuardHome#4911] [Q&A]
I have AGH setup on a VPS with wildcart SSL using `certbot certonly --manual --preferred-challenges=dns --preferred-chain="ISRG Root X1"`
DoH works fine on iOS using the profile generator on setup page but DoT doesn't. Clueless as to how to debug, port 853 on the VPS is open and 
```
lsof -i :853
COMMAND    PID USER   FD   TYPE DEVICE SIZE/OFF NODE NAME
AdGuardHo 1294 root   22u  IPv4  24772      0t0  TCP amsterdam:domain-s (LISTEN)
```
tells that 853 is working. 
ANy help please.

**Comment by ainar-g** [comment]
It's weird, because generally if DoH works then DoT should work as well.  As for some pointers:

1.  Make sure that the domain name that you've used for the certificate is resolvable.

2.  Make sure that you've entered the domain name on the *Settings → Encryption settings* page.

3.  Enable the [verbose logging][1] and look for errors.


[1]: https://github.com/AdguardTeam/AdGuardHome/wiki/FAQ#verboselog

**Reply by bloodyburger** [reply]
Positive for (1) and (2), I will try verbose log and see what turns up
